### PR TITLE
Declare dependencies

### PR DESCRIPTION
**Pull request by nharsch**

### CODE REVIEW

**Comment by bufke** [comment]
Read [this](https://packaging.python.org/requirements/#install-requires-vs-requirements-files) and resubmit. You don't want to force people to use specific versions.

**Comment by bufke** [comment]
Also since this is 7 commits to change one thing - squashing git commits would be nice.

**Comment by nharsch** [comment]
@bufke Thanks, dependencies have either lower bounds on versions or none at all. Squashed commits.

File: setup.py [hunk 1]
```diff
@@ -23,5 +23,13 @@
     ],
     install_requires=[
         'wagtail>=1.0.0',
+        'django-hashers-passlib',
+        'django-storages>=1.4.1',
+        'boto3',
```
Comment (bufke) [inline setup.py]: Why are we requiring all this? The entire world is not using AWS. Nor using contrib comments nor longerusername.

Please think through which packages are actually required and which are optional. You can use [extras](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) in setup.py or just document which packages may be useful.

Comment (nharsch) [reply setup.py]: removed the unnecessary dependencies

File: setup.py [hunk 2]
```diff
@@ -23,5 +23,9 @@
     ],
     install_requires=[
         'wagtail>=1.0.0',
+        'django-contrib-comments>=1.6.1',
+        'django-comments-xtd>=1.5.3',
```
Comment (bufke) [inline setup.py]: Why this one?

Comment (nharsch) [reply setup.py]: Both libs are imported at the top of `blog/management/commands/wordpress_to_wagtail.py`

Comment (bufke) [reply setup.py]: Just note that in the readme. I would suspect 1% of people are using that importer.

Comment (nharsch) [reply setup.py]: Called out in README and removed from setup.py

**Pull request merged**